launcher/container_runner.go

```diff
@@ -86,7 +87,7 @@ func fetchImpersonatedToken(ctx context.Context, serviceAccount string, audience
 func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.Token, launchSpec spec.LaunchSpec, mdsClient *metadata.Client, tpm io.ReadWriteCloser, logger *log.Logger) (*ContainerRunner, error) {
 	image, err := initImage(ctx, cdClient, launchSpec, token, logger)
 	if err != nil {
-		return nil, err
```

Comment:
Since the default is a non-retryable error, is there any reason to annotate all the errors?

it kind of adds bloat. The only reason I see for keeping NonRetryableError around is either to ensure it doesn't get wrapped as a RetryableError or future-proofing

Reply:
Make sense, I was thinking there could be different use caes, but seems like for now. I will just keep the retryable error.